docs/CODEOWNERS.md

```diff
@@ -0,0 +1,502 @@
+- that person is able to enforce consistency in the area
+- that person may be available for giving help in the area
+- that person has push access on the repository
+
+Being a maintainer does not mean the following:
+- that person is dedicated to the area
+- that person is working full-time on the area/on ReactOS
+- that person is paid
+- that person is always available
+
+**We have no supported (paid) areas in ReactOS.**
+
+When submitting a pull request on GitHub and looking for reviewers, look at that file and ask for a review from some of the people listed in the matching area, also, assign the pull request to the M person. Don't ask for a review from all the listed reviewers. Also, when submitted a pull request on GitHub, rules defined in [CONTRIBUTING.md](CONTRIBUTING.md) apply. And if the maintainer is not available and reviewers approved the pull request, developers feeling confident can merge the pull request. Note that reviewers do not necessarily have push access to the repository. When submitting a bug report on Jira, if you want to be sure to have a developer with skills in that area, write @nick from M people.
+
+There should be one and only one primary maintainer per area.
+
+In case of 3rd party code (also referred as upstream), the maintainer is responsible of updating periodically the source code and of managing local patches. He is not here to upstream code on your behalf. As responsible, he may refuse a local patch if you did not try to upstream your changes.
+
+If you want to get listed in this file, either put yourself in the file and push it, or open a pull request. You can also ask a person who has push access to add yourself.
+
+This file uses GitHub's format for specifying code owners.
+- Lines starting with # are comment lines.
+- All other lines specify a path / file (wildcards allowed) followed by the GitHub user name(s) of the code owners. See https://help.github.com/en/articles/about-code-owners
+
+## ReactOS Components
+
+### 3rd Party File Format Libraries
+Maintainer: @ThFabba
+Reviewers: None
+Status: Upstream
+Comments: See media/doc/3rd Party Files.txt
+
+|Component                               |Code Owner(s)|
+|----------------------------------------|-------------|
+|/dll/3rdparty/libjpeg/                  |@ThFabba     |
+|/dll/3rdparty/libpng/                   |@ThFabba     |
+|/dll/3rdparty/libtiff/                  |@ThFabba     |
+|/dll/3rdparty/libxslt/                  |@ThFabba     |
+|/sdk/include/reactos/libs/libjpeg/      |@ThFabba     |
+|/sdk/include/reactos/libs/libmpg123/    |@ThFabba     |
+|/sdk/include/reactos/libs/libpng/       |@ThFabba     |
+|/sdk/include/reactos/libs/libtiff/      |@ThFabba     |
+|/sdk/include/reactos/libs/libxml/       |@ThFabba     |
+|/sdk/include/reactos/libs/libxslt/      |@ThFabba     |
+|/sdk/lib/3rdparty/libmpg123/            |@ThFabba     |
+|/sdk/lib/3rdparty/libsamplerate/        |@ThFabba     |
+|/sdk/lib/3rdparty/libxml2/              |@ThFabba     |
```

Comment:
Does it still assign code owner reviewers to the PRs correctly with this file being markdownized?

Reply:
I copied the information that was in the original codeowners file by hand into markdown tables. It's possible that I made a mistake, but the information should be the same as it is in master currently.

Reply:
The question is: does it still work with GitHub? This file is used to automatically assign reviewers.